crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -408,17 +408,27 @@ fn random1D(s: f32) -> f32 {
     return fract(sin(s * 12.9898) * 43758.5453123);
 }
 
+// Hashed Alpha (https://casual-effects.com/research/Wyman2017Hashed/index.html)
+fn hash2D(in: vec2<f32>) -> f32 {
+    return fract(1.0e4 * sin(17.0 * in.x + 0.1 * in.y) * (0.1 + abs(sin(13.0 * in.y + in.x))));
+}
+
+fn hash3D(in: vec3<f32>) -> f32 {
+    return hash2D(vec2<f32>(hash2D(in.xy), in.z));
```

Comment:
Not familiar with anisotropic hashing (kinda new to this). The paper did mention something about adding noise when changing LOD, but I didn't bother with it right now.

Reply:
Ah, you mean why does it repeat. Well, it is to keep the range within 0 and 1 (since that is what the alpha values are between). Not sure if there is an isotropic hash function with the same properties (constrained range and normal distribution(?)). This is what the authors used, so I stayed with it.